docs/docs/docs/05-utils/pack-up/00-intro.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Introduction
+tags:
+  - pack-up
+  - bundling
+  - packages
+---
+
+## What is pack-up?
+
+Pack-Up is primarily a CLI interface to develop & build CJS/ESM interopable packages. It's built on top of [vite](https://vitejs.dev/),
+but also utilises [esbuild](https://esbuild.github.io/) for certain parts. It also offers a node API that can be used
+to build other tools on top of it.
+
+It's designed to have as little dependencies as possible making it easier to react to changes in the ecosystem as and when we need too.
+The configuration format at present is limited and is expected to grow with real-world requirements.
+
+## Why do we need it?
+
+Creating packages that work in both CJS and ESM environments is a pain. Strapi also has it's own plugin system that realistically
+would benefit from having that code bundled for both the client and server packages of said plugins. Initially we wanted to keep
+this inside the `@strapi/strapi` package but realised soon after that this package was required to be more low level than the former.
```

Comment:
I would have an aditional question, Who will be using this?

At which point of strapi development would you use this? 
Will this be used automatically when publishing and building strapi?

Reply:
updated ✅